packages/twenty-front/src/modules/action-menu/actions/display/components/ActionDropdownItem.tsx

```diff
@@ -1,17 +1,30 @@
 import { ActionDisplayProps } from '@/action-menu/actions/display/components/ActionDisplay';
 import { getActionLabel } from '@/action-menu/utils/getActionLabel';
+import { useNavigate } from 'react-router-dom';
+import { isDefined } from 'twenty-shared/utils';
 import { MenuItem } from 'twenty-ui/navigation';
 
 export const ActionDropdownItem = ({
   action,
+  onClick,
+  to,
 }: {
   action: ActionDisplayProps;
+  onClick?: (event?: React.MouseEvent<HTMLElement>) => void;
```

Comment:
We shouldn't put event here since it's not forwarded

Also would be good to use a handleMenuItemClick